dev/NavigationView/NavigationView.cpp

```diff
@@ -155,6 +155,8 @@ void NavigationView::UnhookEventsAndClearFields(bool isFromDestructor)
     m_topNavOverflowItemsRepeaterElementClearingRevoker.revoke();
     m_topNavRepeaterOverflowView.set(nullptr);
 
+    m_topNavOverflowItemsCollectionChangedRevoker.revoke();
```

Comment:
I noticed there is a `m_footerItemsCollectionChangedRevoker` revoker which listens to collection changes to the NavigationView.FooterItems collection:  https://github.com/microsoft/microsoft-ui-xaml/blob/c7b0ad8148ff53508349921e0b50289aad4de576/dev/NavigationView/NavigationView.cpp#L751

Is there a reason this revoker is not revoked inside the `UnhookEventsAndClearFields()` method?

Reply:
This sounds like a bug to me, we should revoke all listeners.

Reply:
I took the liberty to revoke this listener in `UnhookEventsAndClearFields()` now as part of this PR.